fontforge/fvimportbdf.c

```diff
@@ -2303,22 +2303,26 @@ int FVImportBDF(FontViewBase *fv, char *filename, int ispk, int toback) {
     int oldenccnt = fv->map->enccount;
 
     eod = strrchr(filename,'/');
```

Comment:
Could we add `pathname`, copy filename up to `eod-1` into it in the first case and `copy(".");` in the second, and free at the bottom? It seems better to leave the caller's string unmolested "as long as we're here".